modules/tinymce/src/core/main/ts/api/NotificationManager.ts

```diff
@@ -155,10 +155,14 @@ const NotificationManager = (editor: Editor): NotificationManager => {
       });
     });
 
-    editor.addShortcut('alt+F12', 'Focus to notification', () =>
-      getTopNotification()
-        .map((notificationApi) => SugarElement.fromDom(notificationApi.getEl()))
-        .each((elm) => Focus.focus(elm)));
+    editor.on('keydown', (e) => {
+      if ((e.altKey) && e.key.toLowerCase() === 'f12') {
```

Comment:
```suggestion
      if (e.altKey && e.key.toLowerCase() === 'f12') {
```

Reply:
This change really needs a test if it's pressed for time log a follow up to add a test.

Reply:
@spocke There is a test case for focusing to notification, it was still using `key code` I updated to use keyboard keypress